little-books/src/App.css

```diff
@@ -0,0 +1,10 @@
+
+.active {
+    color: red;
```

Comment:
Have colors in a separate scss file in HEX formate and use it in file

Reply:
resolved